docs/get-started/installer.rst

```diff
@@ -7,9 +7,9 @@ Requirements
 * Windows/Linux/MacOS Computer
 * DCC-EX Compatible motor shield / motor controller board
 * DCC-EX Compatible Microcontroller (Arduino Mega/Uno)
-* USB cable to connect your computer to the Microcontroller Trains
```

Comment:
Hello,
The USB cable doesn't connect to the trains, but did this mean to say something else?
Like Train Controller maybe? Or was left over from an earlier wording?
Cheers, Colin.

Reply:
LOL. So glad someone is going through all this word by word.